scrapers/MetArtNetwork.yml

```diff
@@ -0,0 +1,27 @@
+name: "MetArtNetwork"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - metartnetwork.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="pull-left custom-photo-movie-description"]
+      $credits: //div[@class="pull-left custom-credits-left"]
+      $script: //script[contains(text(),'INITIAL_STATE')]/text()
+    scene:
+      Title: $info/h3/a/@title
+      Details: $info//p/text()
+      Date:
+        selector: $credits//li[span/text()='Released:']/text()
+        parseDate: Jan 2, 2006
+      Performers:
+        Name: $credits//li[span/text()='Cast:']//a/text()
+      Tags:
+        Name: //div[@class="tag-container"]//a/text()
+      Image: //div[@class="custom-movie-preview"]//img/@src
+      Studio:
+        Name: //meta[@property="og:site_name"]/@content
```

Comment:
For the studio i couldn't find a scene that matches but neither did i find another way as it doesn't seem to be present in the page. Searching for the title also has too many results to bother with (common titles) .